src/screens/Order/OrderUIUtils.res

```diff
@@ -427,7 +427,7 @@ let getOrdersList = async (
   try {
     let ordersUrl = getURL(~entityName=ORDERS, ~methodType=Post)
     let res = await updateDetails(ordersUrl, filterValueJson->JSON.Encode.object, Post)
-    let data = res->getDictFromJsonObject->getArrayFromDict("data", [])
+    let data = res->LogicUtils.getDictFromJsonObject->LogicUtils.getArrayFromDict("data", [])
```

Comment:
can u open `LogicUtils` at top

Reply:
This change is not a part of this PR, done in next PR.